rust/lon/src/cli.rs

```diff
@@ -65,6 +65,10 @@ enum Commands {
     Bot {
         #[clap(subcommand)]
         commands: BotCommands,
+        #[arg(long, default_value_t = false)]
+        detailed: bool,
```

Comment:
I'd call this `list_revs` or `list_commits` because `detailed` could mean anything and it's not immediately clear.

Then we can also get rid of the `num_commits` variable and just get that information via this variable (e.g. `LON_LIST_COMMITS = 50`). We can then either force the user to always specify a number of commits they want to see or allow the variable to also be set to true and then use the default. This becomes easier when we read this setting via an env var (see the other comment).